contracts/lease/src/contract/endpoins.rs

```diff
@@ -69,7 +69,7 @@ pub fn migrate(deps: DepsMut<'_>, _env: Env, _msg: MigrateMsg) -> ContractResult
     #[cfg(not(feature = "migration"))]
     let resp = versioning::update_software(deps.storage, version!(CONTRACT_STORAGE_VERSION))
         .and_then(response::response);
-    resp.or_else(|err| log_error(err, deps.api))
+    resp
```

Comment:
what's the motivation ?

Reply:
That must have happened during the rebase as it was really old at this point.